_includes/archive-single.html

```diff
@@ -10,6 +10,12 @@
   {% assign title = post.title %}
 {% endif %}
 
+{% if show_excerpts == false %}
+  {% assign show_excerpt = false %}
```

Comment:
Can we be consistent with plural form here? I'd go with `show_excerpts`. Same for 2 more occurrences below.

Reply:
Sure, as long as you don't mind overshadowing the original variable `show_excerpts`.

Reply:
Hmmm, looks like there's no need for `show_excerpt` at all. Just use `show_excerpts != false` below and it'll be fine.

To be honest, I'm kind of puzzled about all these variables now.

Reply:
Yes, good idea.